src/clr-angular/popover/tooltip/tooltip-trigger.ts

```diff
@@ -6,7 +6,7 @@
 import { Directive, HostListener } from '@angular/core';
 import { IfOpenService } from '../../utils/conditional/if-open.service';
 
-@Directive({ selector: '[clrTooltipTrigger]', host: { '[attr.tabindex]': '0' } })
+@Directive({ selector: '[clrTooltipTrigger]', host: { '[attr.tabindex]': '0', '[class.tooltip-trigger]': 'true' } })
```

Comment:
This class looks like a static class. Why just use `class: "tooltip-trigger"`

Reply:
I don't do that because webstorm syntax highlighting fails on that and I find it very annoying.

Reply:
And more importantly that would override the classes used by the consumer, etrase them to replace them with `tooltip-trigger`.